chart/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -172,6 +172,8 @@ spec:
               value: {{ .Values.gardener.project }}
             - name: APP_GARDENER_KUBECONFIG_PATH
               value: {{ .Values.gardener.kubeconfigPath }}
+            - name: APP_GARDENER_KUBERNETES_VERSION
```

Comment:
are you sure it is APP_GARDENER_KUBERNETES_VERSION ?

Reply:
thanks, good catch 👍